lib/es-models/bib.js

```diff
@@ -10,14 +10,15 @@ const { titleSortTransform } = require('../utils/title-sort-transform')
 const { lookup } = require('../utils/lookup')
 const { getBibCallNum } = require('../utils/get-bib-call-num')
 const { unique, removeTrailingElementsMatching, sortByAsyncSortKey } = require('../utils')
-const { subjectLiteralFromSubfieldMap } = require('../utils/subject-formatter')
 const {
   normalizeAuthorName,
   withoutDates,
   nameIsRedundant,
   trimTrailingPeriod
 } = require('../utils/author-names')
 
+const { firstSubfields, lastSubfields, subjectLiteralFromSubfieldMap } = require('@nypl/browse-term')
```

Comment:
Looks like the npmjs hosted package is ahead of the git repo?

Reply:
Just updated.